tools/IoTEdgeDevOps/DevOpsLib/AgentMatrix.cs

```diff
@@ -7,67 +7,60 @@ namespace DevOpsLib
 
     public class AgentMatrix
     {
-        class CapabilityKeys
-        {
-            public const string AgentOS = "Agent.OS";
-            public const string AgentOSArchitecture = "Agent.OSArchitecture";
-            public const string AgentOSName = "agent-os-name";
-            public const string AgentOSBits = "agent-osbits";
-            public const string RunnerOSName = "runner-os-name";
-        }
+        
 
         readonly IList<AgentDemandSet> rows = new List<AgentDemandSet>()
         {
             new AgentDemandSet(
                 "Linux AMD64",
                 new HashSet<AgentCapability>
                 {
-                    new AgentCapability(CapabilityKeys.AgentOS, "Linux"),
-                    new AgentCapability(CapabilityKeys.AgentOSArchitecture, "X64"),
+                    new AgentCapability(AgentCapabilityKeys.AgentOS, "Linux"),
+                    new AgentCapability(AgentCapabilityKeys.AgentOSArchitecture, "X64"),
```

Comment:
For `AgentOS`, please consider using `OSPlatform.Linux` 
https://docs.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.osplatform?view=netframework-4.8

For `AgentOSArchitecture`, plase consider using `Architecture.X64`
https://docs.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.architecture?view=netframework-4.8

Reply:
For OS platform, there is a value "Windows_NT" report from vsts agent, which can't match in System.Runtime.InteropServices.OSPlatform.  I will add TODO to see if we should create our own OS platform enum as we are going to support Yocto very soon.